hudi-common/src/main/java/org/apache/hudi/common/table/log/HoodieLogFormatWriter.java

```diff
@@ -62,15 +61,14 @@ public class HoodieLogFormatWriter implements HoodieLogFormat.Writer {
       Short replication,
       Long sizeThreshold,
       String rolloverLogWriteToken,
-      LogFileCreationCallback fileCreationHook) {
+      LogFileCreationCallback fileCreationCallback) {
     this.fs = fs;
     this.logFile = logFile;
     this.sizeThreshold = sizeThreshold;
     this.bufferSize = bufferSize;
     this.replication = replication;
     this.rolloverLogWriteToken = rolloverLogWriteToken;
-    this.fileCreationHook = fileCreationHook;
-    addShutDownHook();
```

Comment:
Do we know why this is added in the first place?

Reply:
I guess it is because on HDFS, when a file inputstream is now shutdown properly, the metadata of that file will be hold for sometime, and if any thread tries to create a file with the same name, the HDFS reports an lease occupied error.